mne/decoding/base.py

```diff
@@ -807,9 +807,9 @@ def get_coef(estimator, attr='filters_', inverse_transform=False):
     Reference
     ---------
     .. [1] Haufe, S., Meinecke, F., Gorgen, K., Dahne, S., Haynes, J.-D.,
-       Blankertz, B., & Biessmann, F. (2014). On the interpretation of weigh
+       Blankertz, B., & Biessmann, F. (2014). On the interpretation of weight
        vectors of linear models in multivariate neuroimaging. NeuroImage, 87,
-       96–110. doi:10.1016/j.neuroimage.2013.10.067
+       96-110. doi:10.1016/j.neuroimage.2013.10.067
```

Comment:
FYI things like this are fine if you add the UTF-8 preamble at the top of the doc and `u"""` instead of `"""` to start the string